projects/ad400x-fmcz/src/ad400x_fmcz.c

```diff
@@ -8,60 +8,57 @@
 #include "xil_printf.h"
 #include "spi_engine.h"
 #include "ad400x.h"
+#include "error.h"
+#include "delay.h"
 
 /******************************************************************************/
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
+#define AD400x_EVB_SAMPLE_NO			1000
 #define AD400X_DMA_BASEADDR             XPAR_AXI_AD40XX_DMA_BASEADDR
-#define AD400X_SPI_ENGINE_BASEADDR      XPAR_SPI_AD40XX_AXI_BASEADDR
+#define AD400X_SPI_ENGINE_BASEADDR      XPAR_SPI_ADC_AXI_REGMAP_BASEADDR
 #define AD400x_SPI_CS                   0
 
 #define SPI_ENGINE_OFFLOAD_EXAMPLE	1
 
+struct spi_engine_init_param spi_eng_init_param  = {
+	.type = SPI_ENGINE,
```

Comment:
This comment is not related to this PR.  
If we select the spi_engine using spi_eng_platform_ops, should we have this type parameter?